src/openforms/contrib/zgw/clients/utils.py

```diff
@@ -0,0 +1,34 @@
+from typing import TypedDict
+
+from django.utils import timezone
+
+import pytz
+
+from api_client import APIClient
+
+TIMEZONE_AMS = pytz.timezone("Europe/Amsterdam")
+
+
+def get_today() -> str:
+    now_in_ams = timezone.make_naive(timezone.now(), timezone=TIMEZONE_AMS)
+    return now_in_ams.date().isoformat()
+
+
+class PaginatedResponseData(TypedDict):
+    count: int
+    next: str
+    previous: str
+    results: list
+
+
+def pagination_helper(client: APIClient, paginated_data: PaginatedResponseData):
```

Comment:
I considered it but left it to inference - we don't know for sure what the type is of the data inside the `results` list, so that would require generics in the type hints, and ideally even a generic for the type of `paginated_data`. I'll probably move this helper into the future PyPI package and then go all out with the typing.